css/about.css

```diff
@@ -0,0 +1,192 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
When you start using more and more CSS files, instead of copying and pasting the reset/normalize CSS you can create a CSS file for it and insert it before calling the pages CSS file.